egs/wsj/s5/steps/nnet3/compute_output.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
+#                2016  Vimal Manohar
+# Apache 2.0.
+
+# This script does forward propagation through a neural network. 
+# LDA features and fMLLR are not supported by this script. Only raw features
```

Comment:
no need to mention the things that are not supported-- they are very obscure by now, so this just confused things.